lib/api/confighandler.go

```diff
@@ -51,6 +51,12 @@ func (c *configMuxBuilder) registerConfigInsync(path string) {
 	})
 }
 
+func (c *configMuxBuilder) registerConfigRequiresRestart(path string) {
+	c.HandlerFunc(http.MethodGet, path, func(w http.ResponseWriter, _ *http.Request) {
+		sendJSON(w, map[string]bool{"config-requires-restart": c.cfg.RequiresRestart()})
```

Comment:
```suggestion
		sendJSON(w, map[string]bool{"requiresRestart": c.cfg.RequiresRestart()})
```

I believe we use camelcase (and can increase brevity)?